pipeline/patch_VAE.py

```diff
@@ -129,23 +194,27 @@ def assemble_VAE(paths):
     with open(os.path.join(summary_folder, '%s_file_paths.pkl' % well), 'wb') as f:
         pickle.dump(fs, f)
 
-    print(f"\tsaving {os.path.join(summary_folder, '%s_static_patches.pt' % well)}")
-    torch.save(dataset, os.path.join(summary_folder, '%s_static_patches.pt' % well))
+    # print(f"\tsaving {os.input_path.join(summary_folder, '%s_static_patches.pt' % well)}")
+    # torch.save(dataset, os.input_path.join(summary_folder, '%s_static_patches.pt' % well))
+
+    print(f"\tsaving {os.path.join(summary_folder, '%s_static_patches.pkl' % well)}")
+    with open(os.path.join(summary_folder, '%s_static_patches.pkl' % well), 'wb') as f:
+        pickle.dump(dataset, f, protocol=4)
 
     # Adjust channel mean/std
     # phase: 0.4980 plus/minus 0.0257
     # retardance: 0.0285 plus/minus 0.0261, only adjust for mean
-    phase_slice = dataset.tensors[0][:, 0]
-    phase_slice = ((phase_slice - phase_slice.mean()) / phase_slice.std()) * 0.0257 + 0.4980
-    retard_slice = dataset.tensors[0][:, 1]
-    retard_slice = retard_slice / retard_slice.mean() * 0.0285
-    adjusted_dataset = TensorDataset(torch.stack([phase_slice, retard_slice], 1))
-    print(f"\tsaving {os.path.join(summary_folder, '%s_adjusted_static_patches.pt' % well)}")
-    torch.save(adjusted_dataset, os.path.join(summary_folder, '%s_adjusted_static_patches.pt' % well))
+    # phase_slice = dataset.tensors[0][:, 0]
+    # phase_slice = ((phase_slice - phase_slice.mean()) / phase_slice.std()) * 0.0257 + 0.4980
+    # retard_slice = dataset.tensors[0][:, 1]
+    # retard_slice = retard_slice / retard_slice.mean() * 0.0285
+    # adjusted_dataset = TensorDataset(torch.stack([phase_slice, retard_slice], 1))
+    # print(f"\tsaving {os.input_path.join(summary_folder, '%s_adjusted_static_patches.pt' % well)}")
+    # torch.save(adjusted_dataset, os.input_path.join(summary_folder, '%s_adjusted_static_patches.pt' % well))
```

Comment:
I decided to isolate a preprocessing function solely for scaling/normalizing, this segment of code will be removed

Reply:
Sorry I just noticed the `z_score` function. In the latest PR there is a new `vae_preprocess` function in `HiddenStateExtractor/vq_vae_supp.py` that could cover this step (and also allow the previously saved models to be used smoothly). Let me know if you need any other functionalities for image preprocessing.